project/Dependencies.scala

```diff
@@ -5,10 +5,11 @@ object Dependencies {
 
   // Version numbers
 
-  val PlayVersion = "2.6.5" // if you update this, you probably need to update the following two
```

Comment:
Unfortunately the `PlayVersion` variable is also present in `docs/build.sbt` so changing the value here also requires changing the value there.